theurgy.lic

```diff
@@ -120,10 +321,39 @@ get_from_container('wine')
 fput 'pour wine on altar'
 put_in_container('wine')
 
+if DRSkill.getxp('Theurgy') <= 24
+  comm_truff
+end
+
 fput "recite Meraud, power the holy fires that unleash my righteous vengeance;Chadatru, guide my sword to swing in justice;Everild, give me the power to conquer my enemies;Truffenyi, let me not lose sight of compassion and mercy;Else, I will become like those I despise;Urrem'tier, receive into your fetid grasp these wicked souls;May the Tamsine's realms never know their evil ways again;May all the Immortals guide your faithful soldier #{checkname}."
-DRCT.walk_to '5988'
-fput 'meditate'
+pause 1
+
+waitrt?
+
+clean_altar
+waitrt?
+
+if $settings.favor_god
+  DRC.bput("pray #{$settings.favor_god}", "#{$settings.favor_god}")
+  waitrt?
+else
+  DRC.bput('pray meraud', 'meraud')
+  waitrt
+end  
+
+move 'go path'
+move 'go hole'
+waitrt?
+move 'east'
+move 'go hole'
+move 'out'
+
+DRCT.walk_to('1230')
```

Comment:
needs to be a populated room - more people the better.  I thought about 851 but 1230 was closer to the altar.